### ISSUE TITLE
Plugin receiving 403 Forbidden errors from Logitech

### ISSUE THREAD

**Issue by klanchman** [klanchman/homebridge-logi-circle-controls#38]
```
[3/22/2022, 7:51:23 PM] [Logi Circle Controls] Refreshing access token. Exp: 2022-03-22T15:18:33.000Z Forced? false
[3/22/2022, 7:51:23 PM] [Logi Circle Controls] Outgoing request: POST https://accounts.logi.com/identity/oauth2/token
[3/22/2022, 7:51:24 PM] [Logi Circle Controls] Incoming response: POST https://accounts.logi.com/identity/oauth2/token - 200
[3/22/2022, 7:51:24 PM] [Logi Circle Controls] Outgoing request: GET https://video.logi.com/api/accessories
[3/22/2022, 7:51:24 PM] [Logi Circle Controls] Incoming response: GET https://video.logi.com/api/accessories - 403
[3/22/2022, 7:51:24 PM] [Logi Circle Controls] Error processing request: HTTPError: Response code 403 (Forbidden)
```

**Comment by klanchman** [comment]
Looks like the minimum app version got bumped up. Sometime I should do #25 to avoid this issue...

**Comment by klanchman** [comment]
Fixed in [v3.0.1](https://github.com/klanchman/homebridge-logi-circle-controls/releases/tag/v3.0.1).
